File: src/algorithm-py/stack.py
from typing import Any

class Stack(object):
  def __init__(self) -> None:
    self.stack = []

  def push(self, data) -> None;
    self.stack.append(data)

  def pop(self) -> Any:
    if self.stack:
      return self.stack.pop()



# question

File: src/algorithm-ts/stack&que/stack.ts
class Stack {
  // オブジェクトを書く。
  data: number[]
  constructor() {
    this.data = [];
  }

  push(num: number) {
    this.data.push(num);
  }

  pop() {
    if(this.data.length) {
      this.data.pop();
    }
  }
}


// Input {'key1': 'value1', 'key2': [1,2,3], 'key3': (1,2,3)} Output True
// Input {`key1`: [`value1`, 'key2': [1,2,3], 'key3': (1,2,3)} Output Fae